index.md

```diff
@@ -48,6 +48,361 @@ def is_even(num):
     return num % 2 == 0
 ```
 
+### E1101: no-member {#E1101}
+
+We expect you to access an object(variable, function...) for an existent member.
+For example, you should not call a variable or function which is not defined.
+In the following example, the Rectangle class does not have the area()
+method(member).
+
+
+```python
+class Rectangle:
+    def __init__(self, width, height):
+        """
+        Initialize the Rectangle.
+
+        @type self: Rectangle
+        @type width: int
+        @type height: int
+        @rtype: None
+        """
+        self.width = width
+        self.height = height
+
+r = Rectangle(5, 4)
+
+# no such member in Rectangle class
+print(r.area())  # Error on this line
+```
+
+### E1102: not-callable {#E1102}
+
+We expect you to call an object only if it has been inferred to a callable object.
+In the following example, x is a variable with value 10. We should not call x()
+ because it is not callable.
+
+```python
+x = 10
+print(x())  # Error on this line
+```
+
+### E1111: assignment-from-no-return {#E1111}
+
+In Python, we expect you to assign to a function call with the infered function
+does return an object. This error is similar to E1128. In the following example,
+f() does not return anything. As a result, x has no value if we assign x f().
+Remember! Whenever you define a function, return some objects otherwise it
+could not be assigned.
+
+```python
+def f():
+    print("s")
+
+def g():
+    x = f()  # Error on this line
+    print(x)
+```
+
+### E1120: no-value-for-parameter {#E1120}
+
+We expect you to pass values for each argument (one for each) in a function.
+In the following example, there should be three values passed to the function
+instead of two. Remember not to leave any argument alone.
+
+```python
+def get_sum(x, y, z):
+    """
+    Return the sum of x, y and z.
+
+    @type x: int
+    @type y: int
+    @type z: int
+    @rtype: int
+    """
+    return x + y + z
+
+get_sum(1, 2) # Error on this line
+```
+
+### E1121: too-many-function-args {#E1121}
+
+In python, we expect you to pass values for each argument (one for each) in a
+function. It is just the opposite to E1121. In the following example, more
+values are passed to the function. Do not forget that each position just need
+one value.
+
+```python
+def get_sum(x, y):
+    """
+    Return the sum of x and y.
+
+    @type x: int
+    @type y: int
+    @rtype: int
+    """
+    return x + y
+
+get_sum(1, 2, 3)  # Error on this line
+```
+
+### E1126: invalid-sequence-index {#E1126}
+
+For your work in python, we expect you to use a sequence indexed with a valid
+type. Valid types are ints, slices, and objects with an __index__ method. For
+ example, inside a list, you should always remember to use the "index" to
+ indicate which element you want.  You should always keep in mind that in
+ csc108/148, a slice index could only be integer.
```

Comment:
It seems like the next error is the one which talks about valid slice indexes, not this one.